builtin/providers/aws/resource_aws_route53_zone.go

```diff
@@ -31,6 +31,12 @@ func resourceAwsRoute53Zone() *schema.Resource {
 				Computed: true,
 			},
 
+			"delegation_set_name_servers": &schema.Schema{
```

Comment:
I originally had it as `nameservers`, but decided it might be better just to be clear where it's actually coming from in the AWS API.